OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/OSOutcomeEventsV2Service.java

```diff
@@ -0,0 +1,21 @@
+package com.onesignal.outcomes;
+
+import com.onesignal.OneSignalAPIClient;
+import com.onesignal.OneSignalApiResponseHandler;
+
+import org.json.JSONObject;
+
+class OSOutcomeEventsV2Service extends OSOutcomeEventsClient {
+
+    OSOutcomeEventsV2Service(OneSignalAPIClient client) {
+        super(client);
+    }
+
+    /***
+     * API endpoint /api/v1/outcomes/outcomes/measure
```

Comment:
Thinking we should use `/api/v1/outcomes/measure_sources` for our "v2" measure endponit.